setup.py

```diff
@@ -48,6 +48,8 @@
         "graphviz>=0.20",
         "matplotlib>=3.0.0",
         "lxml>=5.3.0",
+        "numba>=0.60.0",
+        "cffi~=1.16.0",
```

Comment:
Do we need the tilde or can we live with anything greater than this?

Reply:
This definitely conflicted with something else, but I don't quite remember what -- I'm testing now with a newer version and it seems to now work fine, so maybe we can live with a higher version